packages/dara-core/tests/python/test_main.py

```diff
@@ -1458,6 +1510,52 @@ async def resolver(ctx: UpdateVariable.Ctx):
         assert res.json() == 'test_current'
 
 
+async def test_calling_action_restores_args():
```

Comment:
New action API was added in 1.4, this test can be removed

Reply:
thanks, removed